source/documentation/questions-and-answers.html.md.erb

```diff
@@ -0,0 +1,141 @@
+---
+title: Questions and Answers | Common Transit Convention API end-to-end service guide
+weight: 6
+---
+
+<%= partial 'partials/google_tag_manager_js' %>
+
+# Questions and Answers
+<!--- Section owner: CTC Traders API --->
+
+
+Here are your questions and our answers from the API call sessions held on 15th and 16th February 2021.
+
+<table>
+  <tr>
+    <th>Your Question</th>
+    <th>Our Answer</th>
+  </tr>
+  <tr>
+    <td>Will there be a filter on XML EORI since we could potentially have several hundred or thousands of enrolments?</td>
+    <td>Yes, we can filter on EORI over a 28 day period but we are currently working on a more suitable filter.</td>
+  </tr>
+  <tr>
+    <td>How long will OAuthTokens be valid for once they are issued?</td>
```

Comment:
For auth related questions, we could link to the user auth pages: https://developer.service.hmrc.gov.uk/api-documentation/docs/authorisation/user-restricted-endpoints